gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -1839,28 +1825,30 @@ gum_exec_block_write_call_invoke_code (GumExecBlock * block,
                                        const GumBranchTarget * target,
                                        GumGeneratorContext * gc)
 {
-#if DEBUG_LEVEL == 10
-  g_print ("gum_exec_block_write_call_invoke_code - enter\n");
-#endif
-
-  gboolean can_backpatch;
-  GumArm64Writer * cw = gc->code_writer;
+  GumArm64Writer * cw;
   gpointer call_code_start;
   GumPrologType opened_prolog;
 
+#if STALKER_DEBUG_LEVEL == 10
+  g_print ("gum_exec_block_write_call_invoke_code - enter\n");
+#endif
+
+  cw = gc->code_writer;
   call_code_start = cw->code;
   opened_prolog = gc->opened_prolog;
 
   gum_exec_block_open_prolog (block, GUM_PROLOG_MINIMAL, gc);
 
-#if DEBUG_LEVEL == 10
-  g_print (
-      "\t> gum_exec_ctx_replace_current_block_with\n\t\t(block->ctx: %p,\n\t\ttarget->absolute_address: %p|%d)\n",
+#if STALKER_DEBUG_LEVEL == 10
+  g_print ("\t> gum_exec_ctx_replace_current_block_with\n");
+  g_print("\t\t(block->ctx: %p,\n\t\ttarget->absolute_address: %p|%d)\n",
       block->ctx, target->absolute_address, target->base);
 #endif
 
-  /* generate code for the target */
-  /* get the target */
+  /* 
+   *  generate code for the target
+   *  get the target 
```

Comment:
Should only be one space between `*` and each line of the comment. My earlier example had an extra space to right-align the numbers, but that's not the case here.